tensorflow/python/ops/linalg_grad.py

```diff
@@ -300,7 +301,7 @@ def _SvdGrad(op, grad_s, grad_u, grad_v):
         "SVD gradient has not been implemented for input with unknown "
         "inner matrix shape.")
 
-  if not op.get_attr("full_matrices") or not op.get_attr("compute_uv"):
+  if not op.get_attr("compute_uv"):
     s, u, v = linalg_ops.svd(a, compute_uv=True, full_matrices=True)
```

Comment:
Remove full_matrices=True here?

Reply:
I kept it this way to preserve existing behavior; my pull request changes only the logic for the case when `compute_uv` is `True`.  However, I think you're right that it could be removed, and `v[..., :, :m]` could be replaced with `v` [here](https://github.com/tensorflow/tensorflow/blob/17e725c0558581cba19bd6c409698b2c3f88efe5/tensorflow/python/ops/linalg_grad.py#L324) and [here](https://github.com/tensorflow/tensorflow/blob/17e725c0558581cba19bd6c409698b2c3f88efe5/tensorflow/python/ops/linalg_grad.py#L328).

Would it make sense for me to make this change in a separate pull request, since it's not directly related to this one?  I can also just add it to this one if you prefer.